### ISSUE TITLE
1.4.1 Use of Colour - G182: Ensuring that additional visual cues are available when text color differences are used to convey information

### ISSUE THREAD

**Issue by josephlabrecque** [dulibrarytech/digitaldu-frontend#150]

**Comment by josephlabrecque** [comment]
Resolution of issue #151 ties into this - but the application doesn't specifically have links buried within paragraphs of text anyway - so this should be good, or better than good.
